### DISCUSSION TITLE
FR: Save epub - provide book name using webpage title

### DISCUSSION THREAD

**Discussion by zeeyado** [plateaukao/einkbro#438] [Ideas]
Currently, when saving a page as epub, both the file and the book name are titled "einbro epub" and the user has to type or paste in the name manually

Is it possible to suggest a filename and book title using the webpage title?

**Comment by BrilliantThings** [comment]
Yes please. This is currently quite a painful process. I’d love this feature.

**Comment by plateaukao** [comment]
@zeeyado , @BrilliantThings 
Good suggestion!
here your are~ f908bc62

**Reply by BrilliantThings** [reply]
Fantastic. Thank you

**Reply by zeeyado** [reply]
Brilliant, thank you. Can confirm it works as expected :)

**Comment by plateaukao** [comment]
By the way, it's possible to add "save epub" function to toolbar in next version.
